app/src/main/java/com/goayo/debtify/view/ShowMembersFragment.java

```diff
@@ -1,36 +1,96 @@
 package com.goayo.debtify.view;
 
+import android.annotation.SuppressLint;
 import android.os.Bundle;
-import android.util.Log;
 import android.view.LayoutInflater;
+import android.view.Menu;
 import android.view.View;
 import android.view.ViewGroup;
 
+import androidx.activity.OnBackPressedCallback;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.databinding.DataBindingUtil;
 import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProviders;
```

Comment:
This relates to all imports and usages of "androidx.lifecycle.ViewModelProviders". 

Is there any drawbacks to using the newer "androidx.lifecycle.ViewModelProvider" package? We've all been using this one, just raising the discussion.